src/components/TextSize.vue

```diff
@@ -22,10 +22,10 @@ export default {
 <style scoped>
 .text-size-control {
   cursor: pointer;
-  font-family: "Noto Serif";
+  font-family: "Noto Serif", serif;
```

Comment:
I've updated the CSS in `explorehomer` so that these values and others are referenced by global variables using `style-resources-loader`. I guess we might need to have a similar conversation about if/how global CSS rules are going to be shared throughout the modules...

Reply:
`!important`? :)